app/src/main/java/com/uc/mymovies/service/MovieService.kt

```diff
@@ -1,29 +1,5 @@
 package com.uc.mymovies.service
 
-import androidx.lifecycle.MutableLiveData
-import com.google.firebase.FirebaseApp
-import com.google.firebase.firestore.FirebaseFirestore
-import com.google.firebase.firestore.FirebaseFirestoreSettings
-import com.uc.mymovies.dto.Movie
-
 class MovieService {
 
-//    var firebase = FirebaseInstance.firebaseInstance
-    private lateinit var firestore : FirebaseFirestore
-
-    init {
-        firestore = FirebaseFirestore.getInstance()
-        firestore.firestoreSettings = FirebaseFirestoreSettings.Builder().build()
-    }
-
-    fun fetchUserMovies(username: String) : MutableLiveData<ArrayList<Movie>> {
-        var movies = firestore?.collection("users").document(username)
-        movies.get().addOnSuccessListener { document ->
-            if (document != null)
-                print(document)
-        }
-
-        return MutableLiveData()
-    }
-
```

Comment:
Moved firestore logic up a layer, it doesn't seem right to build one in the data layer, but I don't really know much about Firebase.